src/Loggers/LogImage.jl

```diff
@@ -51,3 +51,18 @@ log_image(logger::TBLogger, name::AbstractString, img::AbstractArray{<:Colorant}
 
 log_image(logger::TBLogger, name::AbstractString, imgArray::AbstractArray, format::ImageFormat; step=nothing) =
     log_keyval(logger, name, TBImage(imgArray, format), step)
+
+"""
+log_image(logger, name, obj; [step=current_step])
+
+Renders the object to PNG and sends it to TensorBoard as an image with tag `name`.
+`showable("image/png", obj)` must be true.
+"""
+function log_image(lg::TBLogger, name::AbstractString, obj; step=nothing)
+    if !showable("image/png", obj)
+        @error "cannot log as an image object $obj"
```

Comment:
FYI, what I meant was

```suggestion
        @error "cannot log as an image object $obj"
        return
```

so that it doesn't crash the whole process. Not sure this is desirable here, though.

Reply:
yeah, of course.